modules/post/windows/gather/bloodhound.rb

```diff
@@ -98,55 +109,49 @@ def run
       fail_with(Failure::Unknown, 'PowerShell is not installed')
     end
 
-    extra_params = ''
-    if datastore['Domain']
-      extra_params += "-Domain #{datastore['Domain']} "
+    extra_params = []
+    [
+      [datastore['Domain'], "-d #{datastore['Domain']}"],
+      [datastore['Stealth'], '--Stealth'],
+      # [datastore['SkipGCDeconfliction'], "-SkipGCDeconfliction"],
+      [datastore['ExcludeDomainControllers'], '--ExcludeDCs'],
+      [datastore['DomainController'], "--DomainController #{datastore['DomainController']}"],
```

Comment:
Just to confirm the options changed from `-` to `--`?

This seems to tie to changes in how `Method` `Download & Disk` are utilizing `extra_params`.

Reply:
previously we had switched `-` to `--` for `disk` method, now we're switching from `--` to `-` on `Download` method.  Simple logic reverse